controllers/v1/observationSubmissionsController.js

```diff
@@ -202,7 +201,6 @@ module.exports = class ObservationSubmissions extends Abstract {
           isAPrivateProgram : solutionDocument.isAPrivateProgram,
           frameworkId: solutionDocument.frameworkId,
           frameworkExternalId: solutionDocument.frameworkExternalId,
-          entityTypeId: solutionDocument.entityTypeId,
```

Comment:
@VISHNUDAS-tunerlabs check with frontend that entityTypeId using anywhere in the app